utils/measure/measure/powermeter/powermeter.py

```diff
@@ -23,3 +23,9 @@ def process_answers(self, answers: dict[str, Any]) -> None:
 class PowerMeasurementResult(NamedTuple):
     power: float
     updated: float
+
+
+class ExtendedPowerMeasurementResult(NamedTuple):
+    power: float
+    voltage: float
+    updated: float
```

Comment:
Prefer to have a single result class with an optional field voltage like this:

```suggestion
class PowerMeasurementResult(NamedTuple):
    power: float
    updated: float
    voltage: Optional[float] = None
```